repos/system_upgrade/el9toel10/actors/mysqlcheck/libraries/mysqlcheck.py

```diff
@@ -0,0 +1,123 @@
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import DistributionSignedRPM
+
+import subprocess
```

Comment:
the use of subprocess is forbidden in the project. use the run function instead:
```
from leapp.libraries.stdlib import api, CalledProcessError,  run
```

catch CalledProcessError and OSError exceptions when calling `run`.  Here is the definition of the function:
* https://github.com/oamg/leapp/blob/52cd433d58c50c972cc38e3010c0b8dedf2b88f4/leapp/libraries/stdlib/__init__.py#L149

You can check other actors which use the run function for a hint.